src/js/static-filtering-parser.js

```diff
@@ -1312,7 +1312,7 @@ Parser.prototype.SelectorCompiler = class {
         this.rePseudoElement = /:(?::?after|:?before|:-?[a-z][a-z-]*[a-z])$/;
         this.reProceduralOperator = new RegExp([
             '^(?:',
-                Array.from(parser.proceduralOperatorTokens.keys()).join('|'),
+                Array.from(parser.proceduralOperatorTokens.keys()).join('|'), // eslint-disable-line indent
```

Comment:
This is the only line left and I've disabled the indentation check here for now. Let me know if you have better ideas.

Reply:
No real need for `Array.from( ...` to be indented, this can be changed.

Reply:
Done.